habitat-lab/habitat/datasets/rearrange/modify_episodes_for_object_rearrange.py

```diff
@@ -113,11 +109,14 @@ def get_obj_rec_cat_in_eps(
     return obj_cat, start_rec_cat, goal_rec_cat
 
 
-def read_obj_category_mapping(filename):
+def read_obj_category_mapping(filename, keep_only_recs=False):
```

Comment:
Can you leave some documentation about what `keep_only_recs` does?

Reply:
Added